### PR TITLE
Upgrade runtime to 21.08

### PR DESCRIPTION
**Pull request by hfiguiere**

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build

**Comment by hfiguiere** [comment]
bot, build

File: org.guitarix.Guitarix.json [hunk 1]
```diff
@@ -23,8 +23,8 @@
         "--share=network",
         /* Allow other instances to see lockfiles */
         "--env=TMPDIR=/var/tmp",
-        "--env=LV2_PATH=/app/extensions/Plugins/lv2",
-        "--env=LADSPA_PATH=/app/extensions/Plugins/ladspa"
+        "--env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2:/app/lib/lv2",
```
Comment (Bleuzen) [inline org.guitarix.Guitarix.json]: just like in other apps, remember that this needs a wrapper script

**Comment by Bleuzen** [comment]
anything else holding it back?

File: org.guitarix.Guitarix.json (hunk 1 above)
Comment (hfiguiere) [reply org.guitarix.Guitarix.json]: yeah I have to fix that.

**Comment by hfiguiere** [comment]
> anything else holding it back?

me testing it. but the fix above first.

**Comment by hfiguiere** [comment]
And we have a new upstream.

**Comment by hfiguiere** [comment]
oh FFS.

**Comment by hfiguiere** [comment]
0.43.1 is out.

**Comment by Bleuzen** [comment]
Tested this build and turns out the "Convolver" effect makes guitarix crash.

**Comment by Bleuzen** [comment]
But this also happens on current master (guitarix 0.42.1, runtime 20.08), so this shouldn't hold it back from merging.

**Pull request merged**